src/mono/wasm/wasm.proj

```diff
@@ -167,7 +199,9 @@
     ],
     "WasmOptConfigurationFlags": [
       { "identity": "WasmOptConfigurationFlags", "value": "$(_WasmOptConfigurationFlags)" }
-    ]
+    ],
+    "EmccExportedFunctions": [@(EmccExportedFunction -> '%22%(Identity)%22', ',')],
+    "EmccExportedRuntimeMethods": [@(EmccExportedRuntimeMethod -> '%22%(Identity)%22', ',')]
```

Comment:
same name change here too.

Reply:
This is for the json. Did you mean to comment for the other file?

Reply:
all, good.